src/react-queries/queries/common.ts

```diff
@@ -67,7 +67,7 @@ export const common = createQueryKeys('common', {
     accountMetadata,
     getPosition,
   }: {
-    accountId: AccountId
+    accountId: AccountId | undefined
```

Comment:
We should never fire this query with an undefined `AccountId` (i.e `skipToken` it in `<TransactionRow />`)

Reply:
Fixed in [99ba774](https://github.com/shapeshift/web/pull/7351/commits/99ba774901225d418f116a94bf723b21ae69987e)